scripts/monitoring/cron-send-project-count.sh

```diff
@@ -0,0 +1,13 @@
+!# /bin/bash
+
+#get the number of projects except which are ours
+projects=$(oca get projects | egrep -Ev 'NAME|default|openshift|openshift-infra' | wc -l)
+
+echo
+echo "Number of projects: $projects "
```

Comment:
Please make the script silent unless we run it with -v or it fails - so we do not pollute the cronjob mails.